ClientSide(FrontEnd)/clay_/src/components/ImageSlider.js

```diff
@@ -0,0 +1,41 @@
+import React, { useState } from 'react';
+import { SliderData } from './SliderData';
+// import { FaArrowAltCircleRight, FaArrowAltCircleLeft } from 'react-icons/fa';
+
+const ImageSlider = ({ slides }) => {
+  const [current, setCurrent] = useState(0);
+  const length = slides.length;
```

Comment:
add default value as 0 to the length variable, if you didn't get anything from slides.length, Code will break;